### ISSUE TITLE
[Bug]: Sites are unreachable

### ISSUE THREAD

**Issue by Shahinyanm** [beyondcode/herd-community#891]
### Platform

macOS

### Operating system version

macOs Sonoma 14.5

### System architecture

Intel (x86)

### Herd Version

1.9.1

### Bug description

Problem Details

	1.	After Upgrading to Laravel Herd 1.9.1:
	•	All previously working sites are no longer accessible.
	•	herd links and herd parked commands show that the sites are linked and parked correctly.
	2.	Reinstallation Attempts:
	•	I tried uninstalling and reinstalling Laravel Herd, but the issue persists.
	3.	DNS Configuration:
	•	The DNS settings are default for macOS.
	•	The nameserver is set correctly.
	4.	Firewall:
	•	The firewall is disabled, but there is no change in behavior.
	5.	Logs:
	•	There are no relevant entries in the logs indicating any issues.

### Steps to reproduce

<img width="1197" alt="Screenshot 2024-07-20 at 23 17 10" src="https://github.com/user-attachments/assets/b1b3be4f-4e16-4be3-93aa-a5ee23f89049">

**Comment by mechelon** [comment]
Can you try to force quit all services (hold the option key while the Herd menubar is open) and share your Nginx/DNSMasq error logs?
![CleanShot 2024-07-20 at 21 19 13@2x](https://github.com/user-attachments/assets/1612cb57-bfa3-4d40-9f0b-b2b75ea846fb)

**Comment by Shahinyanm** [comment]
<img width="1728" alt="Screenshot 2024-07-20 at 23 33 08" src="https://github.com/user-attachments/assets/44e7e39a-2d17-42fa-a2fb-d522f65049f2">


Dnsmasq now is using as i have installed localy , before there weren't any errors

**Comment by indrapalijama** [comment]
any updates on this one?

**Comment by Shahinyanm** [comment]
> any updates on this one?

Yup,  I have fixed,  The first link was working automatically, but after the upgrade, it  need to add 127.0.0.1 site.test to /etc/hosts

 After that everything working correctly

**Comment by mechelon** [comment]
When I understand that correctly, you have installed another instance DNSMasq in parallel to Laravel Herd?
It only supports using the instance that comes with it, then you also don't have to modify the hosts file.

**Comment by Shahinyanm** [comment]
> When I understand that correctly, you have installed another instance DNSMasq in parallel to Laravel Herd? It only supports using the instance that comes with it, then you also don't have to modify the hosts file.

Nope, i didn't
 I only upgrade Laravel herd from 1.4.1 - to 1.9.1 , restart macos, and then no sites was reached 
 
 After ton of manipulations , delete and install, tried to write manually site  in hosts , and it works,

**Comment by sschlein** [comment]
There's definitely something running on the same port that dnsmasq uses and so Herd can't start the version that's shipping with it. If you can solve this and stop the other program from blocking it, you'll be good with Herd

**Comment by indrapalijama** [comment]
> There's definitely something running on the same port that dnsmasq uses and so Herd can't start the version that's shipping with it. If you can solve this and stop the other program from blocking it, you'll be good with Herd

yes, i noticed that my dnscrypt-proxy is using the host (127.0.0.1) as dnsmasq so it's not working, do you have any idea how to run it side by side?

**Comment by sschlein** [comment]
You can't – that's not supported by Herd

**Comment by Shahinyanm** [comment]
> There's definitely something running on the same port that dnsmasq uses and so Herd can't start the version that's shipping with it. If you can solve this and stop the other program from blocking it, you'll be good with Herd

I have checked , there is only one process for dnsmasq and it's started by herd,

**Comment by victor-falcon** [comment]
I had the same problem.

Now it's working after adding 127.0.0.1 site.test to my `/etc/hosts` but I think there is something wrong with DNSMasq, don't know why is this issue closed.

**Comment by ataldev** [comment]
> I had the same problem.
> 
> Now it's working after adding 127.0.0.1 site.test to my `/etc/hosts` but I think there is something wrong with DNSMasq, don't know why is this issue closed.

But this is not the solution! it's supposed to work without this. Herd should handle this by itself.

**Comment by mechelon** [comment]
The initial issue was closed because Herd does not support parallel use of two Dnsmasq instances. Feel free to open another issue with your specific problem.
(Modifying your hosts file should not be necessary.)
